fs/proc/ish.c

```diff
@@ -55,6 +55,10 @@ static int proc_ish_show_colors(struct proc_entry *UNUSED(entry), struct proc_da
 }
 
 static int proc_ish_show_documents(struct proc_entry *UNUSED(entry), struct proc_data *buf) {
+    if (get_documents_directory == NULL) {
+	proc_printf(buf, "\n");
+	return 0;
```

Comment:
Please fix indentation (is this perhaps using tabs instead of spaces or something?

Reply:
Yes, tabs.  You may see these in C code occasionally.  :)

You might want to add some style guidance to the docs, or enforce with `clang-format`, if it's important to you.

Also fixed in #2387

Reply:
The problem with it is that github renders tabs as four spaces, while your editor probably used eight... thus are the hazards of mixing indentation types.